docs/targets/twilio.md

```diff
@@ -1,119 +1,50 @@
 
 ## Prerequisites
 
-A Twilio account is required to run this target:
-
-* Register a Twilio account
-* Purchase a phone number with
-* Retrieve from Twilio Dashbard Account SID
-* Retrieve from Twilio Dashbard Auth Token
-
-## Controller Deployment
-
-### Kubernetes manifests
-
-// TODO use our images
-
-### From code
-
-You can use the [ko](https://github.com/google/ko) tool to compile and deploy from source.
-
-```console
-ko create -f ./config
-```
-
-## Creating a Twilio Target
-
-Once the Twilio Target Controller has been deployed along all other needed assets are present we can create integrations by adding TwilioTargets objects.
-
-```yaml
-apiVersion: targets.triggermesh.io/v1alpha1
-kind: TwilioTarget
-metadata:
-  name: <TARGET-NAME>
-spec:
-  defaultPhoneFrom: "<PHONE-FROM>"
-  defaultPhoneTo: "<PHONE-TO>"
-  sid:
-    secretKeyRef:
-      name: "<YOUR-SID-SECRET>"
-      key: "<YOUR-SID-SECRET-KEY>"
-  token:
-    secretKeyRef:
-      name: "<YOUR-TOKEN-SECRET>"
-      key: "<YOUR-TOKEN-SECRET-KEY>"
-```
+1. [Twilio][try] account with access to the Account SID & Auth Token.
+1. [Phone Number][pn].
 
-Although `defaultPhoneFrom` is not mandatory you will usually configure it matching the phone number you have purchased at Twilio.
+## Deploying an Instance of the Target
 
-On the other hand `defaultPhoneTo` wont be usually informed unless you want all messages to address the same phone number by default.
+Open the Bridge creation screen and add a Target of type `Twilio`.
 
-Both configurations can be overrided at every Cloud Event message received at the Target.
+![Adding a Twilio Target](../images/twilio-target/create-bridge-1.png)
 
-Refer to [Twilio docs for number formating](https://www.twilio.com/docs/lookup/tutorials/validation-and-formatting?code-sample=code-lookup-with-international-formatted-number).
+In the Target creation form, give a name to the event Target and add the following information:
 
-### Status
+* **Default source phone number**: Defines a default sending phone. This is not mandatory; however, it is usually configured to the [phone number][pn] purchased at Twilio.
+* **Default destination**: If informed, all messages will default to this phone number.
+* **SID Secret**: Reference to a [TriggerMesh secret][tm-secret] containing a SID token for authenticating requests.
+* **Token Secret**: Reference to a [TriggerMesh secret][tm-secret] containing an API token for authenticating requests.
 
-TwilioTarget requires two Secrets to be provided for SID and Token, once they are present it will create a Knative Service. Controller logs and events can provide detailed information about the process. A Status summary is added to the TwilioTarget object informing of the all conditions that the target needs.
+Both the **Default source phone number** and **Default destination** configurations can be overrided by any [CloudEvent][ce] message received at the Target.
```

Comment:
overwritten ? overriden ? cc/ @cab105

Reply:
Should be `may be overridden`.  Also, `received by the Target`.